pages.id/common/arduino.md

```diff
@@ -0,0 +1,36 @@
+# arduino
+
+> Arduino Studio - Sebuah alat pengembangan piranti lunak (IDE) bagi platform Arduino.
+> Informasi lebih lanjut: <https://github.com/arduino/Arduino/blob/master/build/shared/manpage.adoc>.
+
+- Bangun piranti lunak dari suatu berkas (sketsa) kode sumber:
+
+`arduino --verify {{jalan/menuju/berkas.ino}}`
+
+- Bangun dan pasang piranti lunak menuju perangkat Arduino:
+
+`arduino --upload {{jalan/menuju/berkas.ino}}`
+
+- Bangun dan pasang piranti lunak menuju suatu perangkat Arduino Nano dengan prosesor Atmega328p yang terhubung dalam port `/dev/ttyACM0`:
+
+`arduino --board {{arduino:avr:nano:cpu=atmega328p}} --port {{/dev/ttyACM0}} --upload {{jalan/menuju/berkas.ino}}`
+
+- Atur `nilai` untuk suatu jenis preferensi/konfigurasi berdasarkan `nama` atau kata kunci:
+
+`arduino --pref {{nama}}={{nilai}}`
+
+- Bangun piranti lunak, kemudian simpan menuju suatu direktori hasil pembangunan, dan gunakan kembali hasil-hasil sebelumnya di dalam direktori tersebut:
+
+`arduino --pref build.path={{jalan/menuju/direktori_hasil_pembangunan}} --verify {{jalan/menuju/bekas.ino}}`
+
+- Simpan segala perubahan pada preferensi/konfigurasi menuju berkas `preferences.txt`:
+
+`arduino --save-prefs`
+
+- Pasang piranti pendukung pengembangan untuk perangkat Arduino berbasis SAM (seperti Arduino Due):
```

Comment:
This one seems to not be equal translation of the English page.
English page:
> Install the latest SAM board:

Google Translate result for your translation:
> Install development support tools for SAM-based Arduino devices (such as Arduino Due):

Reply:
The "SAM board" here should refer to the necessary SDKs to build the code/sketch to SAM-based boards.

In the GUI version of Arduino, this is usually done by going through the Tools menu > Board > Boards Manager to install such tools.

!["Install the SAM Board" menu on the Arduino GUI](https://github.com/tldr-pages/tldr/assets/17312341/51269817-04db-4583-a4be-8512bca19281)